packages/tailwind/integrations/vite/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "vite-with-tailwind",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "preinstall:tailwind": "cd ../../ && turbo build && yalc installations clean @react-email/tailwind && yalc publish",
+    "preinstall": "npm run preinstall:tailwind && yalc add @react-email/tailwind",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "react": "^18.2.0",
+    "@react-email/components": "^0.0.15",
+    "@react-email/tailwind": "file:.yalc/@react-email/tailwind",
```

Comment:
Is this correct?

Reply:
Yup, if we test importing from the workspace directly the testing won't be as conclusive as this one, a lot of issues would get by without noticing from my experience.